src/lib.rs

```diff
@@ -440,4 +440,78 @@ impl TxBuilder {
     }
 }
 
+struct BumpFeeTxBuilder {
+    txid: String,
+    fee_rate: f32,
+    rbf: Option<RbfValue>,
+    allow_shrinking: Option<String>,
```

Comment:
Small nit: the order of the `rbf` and `allow_shrinking` parameters are switched between the struct declaration and the different methods (sometimes rbf is first sometimes second).

Reply:
Good catch, for readability/maintainability it does matter. Fixed.